Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php

```diff
@@ -139,7 +139,7 @@ private function isValidControl(Tokens $tokens, Token $docsToken, $controlIndex)
         $endIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
         $docsContent = $docsToken->getContent();
 
-        for ($index = $index + 1; $index < $endIndex; ++$index) {
+        for (++$index; $index < $endIndex; ++$index) {
```

Comment:
One way or another, less readable for me.
- set initial value of iterator
- test iterator
- change iterator

now the first step is changing too